client/app/config/all.js

```diff
@@ -142,6 +142,9 @@ angular.module("managerApp").constant("LANGUAGES", {
     })
     .constant("REDIRECT_URLS", {
         billing: "https://www.ovh.com/manager/dedicated/index.html#/billing/history",
+        consumptionsTelephony: "https://www.ovh.com/manager/dedicated/index.html#/billing/consumptionsTelephony",
```

Comment:
This section will be removed from billing section of dedicated manager because everything can be found in TELECOM manager. You can remove it :-)

Reply:
Ok, removed